### PR TITLE
Make Fe heartbeat timeout same to be/broker

### PR DESCRIPTION
**Pull request by gengjun-git**

### CODE REVIEW

**Review by imay — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java [hunk 1]
```diff
@@ -295,7 +295,9 @@ public HeartbeatResponse call() {
             String url = "http://" + fe.getHost() + ":" + Config.http_port
                     + "/api/bootstrap?cluster_id=" + clusterId + "&token=" + token;
             try {
-                String result = Util.getResultForUrl(url, null, 2000, 2000);
+                String result = Util.getResultForUrl(url, null,
+                                                     FeConstants.heartbeat_interval_second * 1000,
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java:299]: The constant variable is configurable ?

File: fe/fe-core/src/main/java/com/starrocks/common/util/Util.java [hunk 2]
```diff
@@ -353,7 +353,7 @@ public static String dumpThread(Thread t, int lineNum) {
     //      Base64.encodeBase64String("user:passwd".getBytes());
     // If no auth info, pass a null.
     public static String getResultForUrl(String urlStr, String encodedAuthInfo, int connectTimeoutMs,
-                                         int readTimeoutMs) {
+                                         int readTimeoutMs) throws  Exception {
         StringBuilder sb = new StringBuilder();
         InputStream stream = null;
```
Comment (chaoyli) [inline fe/fe-core/src/main/java/com/starrocks/common/util/Util.java:358]: why to change the heartbeat timeout to be configurable?

File: fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java (hunk 1 above)
Comment (gengjun-git) [reply fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java:299]: I have changed the title

File: fe/fe-core/src/main/java/com/starrocks/common/util/Util.java (hunk 2 above)
Comment (gengjun-git) [reply fe/fe-core/src/main/java/com/starrocks/common/util/Util.java:358]: not configurable, just same to be/broker heartbeat timeout

File: fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java (hunk 1 above)
Comment (kangkaisen) [reply fe/fe-core/src/main/java/com/starrocks/system/HeartbeatMgr.java:299]: Why FE heartbeat timeout need to same with BE or Broker? Why the heartbeat timeout isn't configurable?

**Pull request closed without merging**